Neos.ContentRepositoryRegistry/Classes/ContentRepositoryRegistry.php

```diff
@@ -115,6 +115,7 @@ public function subgraphForNode(Node $node): ContentSubgraphInterface
     }
 
     /**
+     * @internal this is a low level extension mechanism and not part of the public API.
```

Comment:
well to retrieve more or less api things like the `ContentStreamPruner` we probably need to declare `buildService` api, and just say creating your own service is not api?